doc/rados/operations/health-checks.rst

```diff
@@ -539,32 +539,33 @@ or delete some existing data to reduce utilization.
 BLUEFS_SPILLOVER
 ________________
 
-One or more OSDs that use the BlueStore backend have been allocated
-`db` partitions (storage space for metadata, normally on a faster
-device) but that space has filled, such that metadata has "spilled
-over" onto the normal slow device.  This isn't necessarily an error
-condition or even unexpected, but if the administrator's expectation
-was that all metadata would fit on the faster device, it indicates
-that not enough space was provided.
+One or more OSDs that use the BlueStore back end have been allocated `db`
+partitions (that is, storage space for metadata, normally on a faster device),
+but because that space has been filled, metadata has "spilled over" onto the
+normal slow device. This is not necessarily an error condition or even
+unexpected behavior, but if the administrator had expected that all metadata
```

Comment:
unexpected behavior but may result in performance degradation.  If the administrator expects that all metadata iwll fit